package/libs/openssl/Config.in

```diff
@@ -313,21 +281,6 @@ config OPENSSL_ENGINE_CRYPTO
 		API modules) for /dev/crypto to show up and use hardware
 		acceleration; otherwise it falls back to software.
 
-config OPENSSL_ENGINE_DIGEST
```

Comment:
Why is the hardware digest option removed? Is it still broken?

Reply:
The option to build without hw accelerated digests was removed in 1.1.0--I had added it back with a patch, since hw digest support was broken before 1.1.1a, but it should be fixed now.